### PR TITLE
[MAISTRA-489] update spec for 0.12.0

### PR DESCRIPTION
**Pull request by bdecoste**

### CODE REVIEW

**Review by brian-avery — APPROVED** [review]

File: istio-proxy/istio-proxy.spec [hunk 1]
```diff
@@ -130,6 +130,10 @@ TEST_ENVOY=false RUN_TESTS=true %{SOURCE2}
 /usr/local/bin/envoy
 
 %changelog
+* Thu Jun 20 2019 William DeCoste <wdecoste@redhat.com>
```
Comment (brian-avery) [inline istio-proxy/istio-proxy.spec:133]: Does it make sense to mention the Istio/Envoy version when specifying a Maistra version upgrade?

**Comment by jwendell** [comment]
Do you have a JIRA number?
Aren't you using/upgrading `sources` file?

**Comment by brian-avery** [comment]
Jira for this should be https://issues.jboss.org/browse/MAISTRA-489

**Comment by jwendell** [comment]
Also, should you update other references to 0.11, e.g. https://github.com/Maistra/rpms/blob/maistra-0.12/istio-proxy/build.sh#L6 ?

**Comment by bdecoste** [comment]
Just updated build.sh. There is no sources file for maistra. I am updating the corresponding product files separately.

**Pull request merged**